### PR TITLE
Wildcards(ForceProcess.md)

### PR DESCRIPTION
**Pull request by Luro223**
Basic info describing wildcards * and ?

### CODE REVIEW

File: Content/ForceProcess.md [hunk 1]
```diff
@@ -9,11 +9,18 @@ _ForceProcess_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md). It spec
    [DefaultBox]
    ForceProcess=iexplore.exe
    ForceProcess=firefox.exe
+   ForceProcess=App*.exe
+   ForceProcess=App?.exe
    [MailBox]
    ForceProcess=outlook.exe
+   ForceProcess=c?ick??.exe
```
Comment (isaak654) [inline Content/ForceProcess.md]:
```suggestion
   ForceProcess=click??.exe
```

In the lines below, there is a specific `click??` reference.

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: Content/ForceProcess.md (hunk 1 above)
Comment (Luro223) [reply Content/ForceProcess.md]: `c?ick??` shows the use of wildcards in the middle

File: Content/ForceProcess.md [hunk 2]
```diff
@@ -9,11 +9,18 @@ _ForceProcess_ is a sandbox setting in [Sandboxie Ini](SandboxieIni.md). It spec
...
    ForceProcess=outlook.exe
+   ForceProcess=cl?cke?.exe
+   
 ```
 
-The example specifies that Internet Explorer (iexplore.exe) and Firefox (firefox.exe) will be forced to run sandboxed in the sandbox _DefaultBox_. Outlook.exe will be forced to run sandboxed in the sandbox _MailBox_.
+- `*` defines any character.
+- `?` defines one character.
+
+The example specifies that Internet Explorer (iexplore.exe), Firefox (firefox.exe), App* (Appga, App03 and etc.). and App? (App1, Appg, Appa and etc.). will be forced to run sandboxed in the sandbox _DefaultBox_. Outlook.exe and click?? (clickor, clickid and etc.). will be forced to run sandboxed in the sandbox _MailBox_.
```
Comment (isaak654) [inline Content/ForceProcess.md]:
```suggestion
The example specifies that Internet Explorer (iexplore.exe), Firefox (firefox.exe), App* (Appga, App03 and etc.). and App? (App1, Appg, Appa and etc.). will be forced to run sandboxed in the sandbox _DefaultBox_. Outlook.exe and cl?cke? (clicker, clicked and etc.). will be forced to run sandboxed in the sandbox _MailBox_.
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Comment by Luro223** [comment]
@isaak654 need review

**Review by isaak654 — APPROVED** [review]

**Pull request merged**